docs/Guides/Contributing.md

```diff
@@ -184,7 +184,9 @@ document, overwriting any text already present, and save it:
 ```
 
 Finally, from the menu bar, select "Terminal > New Terminal" to open a new terminal
-in the editor. Run `npm i` to install the Fastify dependencies.
+in the editor. Run `npm i --package-lock` to install the Fastify dependencies
+(the `--package-lock` flag is required for the cross-env dependency to be
+installed).
```

Comment:
How can this be true? Why would a package need that switch in order to be installed?

Reply:
I restored back this line because it worked without the flag. I think there was something wrong with my previous configuration